src/Pimple/KafkaSchemaRegistryApiClientProvider.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Jobcloud\KafkaSchemaRegistryClient\Pimple;
+
+use Buzz\Client\Curl;
+use Jobcloud\KafkaSchemaRegistryClient\ErrorHandler;
+use Jobcloud\KafkaSchemaRegistryClient\HttpClient;
+use Jobcloud\KafkaSchemaRegistryClient\KafkaSchemaRegistryApiApiClient;
+use LogicException;
+use Nyholm\Psr7\Factory\Psr17Factory;
+use Pimple\Container;
+use Pimple\ServiceProviderInterface;
+use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
+
+class KafkaSchemaRegistryApiClientProvider implements ServiceProviderInterface
+{
+
+    public const ROOT = 'kafka_schema_registry';
```

Comment:
```suggestion
    public const CONTAINER_KEY = 'kafka_schema_registry';
```

Reply:
hahhaa, I wanted to name it `CONTAINER_KEY` but thought you said root in our conversation so I named root :D Will change.